### DISCUSSION TITLE
using comments on table and view level as well as column level

### DISCUSSION THREAD

**Discussion by arthurvetter** [ibis-project/ibis#8347] [Q&A] [answered]
Hi all,

is it possible to add comments to views/tables and columns via a table function?

I did not find anything in the documentation and it is also not part of the ANSI SQL Standard i think. However, many backends support commenting on tables/views and columns.

The only way of achieving it, is using the raw_sql table method i guess?

Thanks in advance

**Comment by cpcloud** [comment, marked as answer]
This is currently not possible, and we don't have plans to support it.
